lint/rule_groups.go

```diff
@@ -47,6 +47,13 @@ func aipCloudGroup(aip int) string {
 	return ""
 }
 
+func aipGURPGroup(aip int) string {
```

Comment:
I'd normally expect this to be aipGurpGroup, but I don't know what Go rules are normally like for acronyms.

Reply:
Yes, golint even complains. :-)